spec/system/page_title_spec.rb

```diff
@@ -10,14 +10,14 @@
 
     it { expect(setting_path('cookie-policy')).to have_page_title 'Cookie policy' }
 
-    it { expect(static_path('accessibility-statement')).to have_page_title 'Accessibility statement' }
```

Comment:
@agoldstone93 the reason the specs should be failing is not because of the `master` env, it should be looking at `demo`. Try populating that then running your specs locally without any override.

Reply:
Yes, you're right. Not sure why I convinced myself it was looking at `master`. I'll do that